articles/connections/passwordless/native-passwordless-universal.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Passwordless Authentication in Native Applications with Universal Login
+---
+# Passwordless Authentication in Native Applications with Universal Login
+
+[Universal Login](/hosted-pages/login) is the only recommended way to set up passwordless authentication in your native application. We recommend using the Auth0 login page for the best experience, security and the fullest array of features.
+
+Once you are done with this guide, the process should be that a user runs across a need to authenticate in your application and the Auth0 SDK redirects them to the login page on the web. At that point, Lock Passwordless will ask them for a phone number or email, whichever type you chose, which they will give, and the prompt will change to ask them for a code which they will receive by the designated metho. Once they enter the code, the transaction will finish and the user will be redirected to your app along with their credentials.
+
+Implementing passwordless in the login page involves just a few steps.
+
+## 1. Set up your connection
+
+To set up the passwordless connection, go to your [Dashboard](${manage_url}/#/connections/passwordless) under **Connections > Passwordless** and turn on either Email or SMS. Also, ensure that you enable the connection for the client you are using. From the connection's settings, you can click the **Apps** tab to toggle on clients that you want to be able to use this connection.
```

Comment:
Simplify the last two sentences with clear instructions `Go to your the **Apps** tab and enable the clients that you want to use this connection`.